.github/workflows/lintercheck.yml

```diff
@@ -0,0 +1,79 @@
+name: Linter check
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - r[0-9]+.[0-9]+
+
+jobs:
+  linter_check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
+          cache: 'pip'
+      - run: pip install yapf==0.30.0
+
+      - name: Check no TORCH_PIN
+        if: github.event_name == 'push' && github.event.ref == 'master'
```

Comment:
Wouldn't this run too late? Basically after the code's already been pushed to master?

Reply:
Yep, but on the other hand this is a copy-paste of the existing logic, nothing's new here...

Reply:
I remembered @ailzhang and I discussed about this long long time ago. On master CI we just caught this error and then someone(most likely me) will revert the pin lol